docs/getting-started/tutorial.md

```diff
@@ -0,0 +1,382 @@
+
+This is what you should see:
+![Blank page](../images/blank-page.png)
+
+You'll notice a few things:
+1. The title and subtitle of the form at the top, which you can change at the top level of the `formConfig`.
+2. A progress bar with 2 segments. Each segment corresponds to a `chapter` of your form; the number of segments to display is determined by the library based on how many `chapter`s you've added to your config.
+3. The "step" of the form you're on out of how many steps there are. Again, steps correspond to the number of `chapter`s, and the current step is the number of the current `chapter`.
+4. The `title` of the `chapter`.
+5. The navigation buttons, "Back" and "Continue".
+
+There's nothing else on this page because we haven't added any questions yet! Let's do that next.
+
+### Step 6: Adding a question
+
+For the purposes of this tutorial, we'll be building a simple address form. The first question when asking for an address is typically the street, so let's start by adding a field for street within the `schema: { properties: {}}` object we just created.
+
+```js
+...
+  schema: {
+    type: 'object',
+    properties: {
+      street: {
+        type: 'string'
+      }
+    }
+  }
+...
+```
+
+Let's see what the form looks like now that we've added this question. The webpack server should have automatically refreshed the page with your form, so go back to your browser to take a look at what's there.
+
+This is what you should see:
+![Street field](../images/street-field.png)
+
+Hooray, a form field has appeared! But how did the library know to render a text input with a label of `street` based on what we added to the `schema`?
+
+The data that the `street` field accepts is `string` data. For the basic data types, an automatic determination is made by the library code on what type of HTML form element to render. `string` data renders a text input, a field with an `enum` property renders a select, and `boolean` data renders checkbox.
+
+The label is taken from the name of the field in the `formConfig`. `street` becomes "street" in our form. But what if we want to change the name that appears to the user? Let's do that next.
+
+### Step 7: Adding the `uiSchema` object.
+
+While the `schema` is the main description of the fields of our form, there are UI-specific things we may want to change about how those fields appear. Remember, the `schema` is mainly just a description of the type of `data` of that field, and not as concerned with how that field is displayed to the user.
+
+That's where `uiSchema` comes in. You can think of the `uiSchema` object as a mirror to the `schema` one. It usually has the same fields as the `schema` object, with different properties within each field to specify UI-specific characteristics of that field.
+
+The `uiSchema` object is added to our `page` at the same level as the `schema` object:
+
+```js
```

Comment:
I'd leave out this code block for the same reason as above, it doesn't add much compared to the one below.